src/main/java/io/vlingo/symbio/store/journal/jdbc/JDBCDispatcherControlDelegate.java

```diff
@@ -146,11 +146,11 @@ private void fail() {
         final String entryType = resultSet.getString(3);
         final int eventTypeVersion = resultSet.getInt(4);
         final String entryMetadata = resultSet.getString(5);
-        final int entryVerion = resultSet.getInt(6); // from E_STREAM_VERSION
+        final int entryVersion = resultSet.getInt(6); // from E_STREAM_VERSION
```

Comment:
@pflueras I don't understand what changed here.

Reply:
It was a typo: entryVerion. Missing 's'.